src/logging.cc

```diff
@@ -1436,7 +1436,7 @@ void LogMessage::SendToLog() EXCLUSIVE_LOCKS_REQUIRED(log_mutex) {
     LogDestination::WaitForSinks(data_);
 
     const char* message = "*** Check failure stack trace: ***\n";
-    if (write(STDERR_FILENO, message, strlen(message)) < 0) {
+    if (write(STDERR_FILENO, message, (unsigned int)strlen(message)) < 0) {
```

Comment:
Changed to #pragma warning( disable: 4267 ) since on Windows strlen returns size_t while write uses unsigned int.